src/audio/dai-zephyr.c

```diff
@@ -76,7 +76,7 @@ int dai_assign_group(struct dai_data *dd, struct comp_dev *dev, uint32_t group_i
 {
 	if (dd->group) {
 		if (dd->group->group_id != group_id) {
-			comp_err(dev, "dai_assign_group(), DAI already in group %d, requested %d",
+			comp_err(dev, "DAI already in group %d, requested %d",
```

Comment:
This is actually a Zephyr feature (CONFIG_LOG_FUNC_NAME_PREFIX) and not added by SOF comp_() macros, so non-Zephyr targets might still benefit from the function name. But alas, this file is Zephyr specific, so no reason to keep the dual names (and we actually save memory and cycles by reducing these).

Reply:
@kv2019i, yes, it is the zephyr prints that does  this, thanks for the clarification.